i18n/zh/docusaurus-plugin-content-docs/current/tutorials/2-start.md

```diff
@@ -266,13 +266,11 @@ export default defineConfig({
   compilation: {
     resolve: {
       alias: {
-        '@/': path.join(process.cwd(), 'src')
+        "@/": path.join(process.cwd(), "src"),
       },
-      externals: [
-        'node:fs'
-      ]
-    }
-  }
+    },
+    externals: ["node:fs"],
```

Comment:
Thanks for your correction, however, I found that in https://github.com/farm-fe/farm/blob/main/packages/core/src/config/types.ts#L120, the exact field is `external`, @ErKeLost take a look?